spring-cloud-dataflow-server-core/src/main/java/org/springframework/cloud/dataflow/server/service/impl/AbstractStreamService.java

```diff
@@ -54,9 +64,60 @@
 	 */
 	protected final StreamDefinitionRepository streamDefinitionRepository;
 
-	public AbstractStreamService(StreamDefinitionRepository streamDefinitionRepository) {
+	/**
+	 * The app registry this controller will use to lookup apps.
+	 */
+	private final AppRegistryCommon appRegistry;
+
+	public AbstractStreamService(StreamDefinitionRepository streamDefinitionRepository, AppRegistryCommon appRegistry) {
 		Assert.notNull(streamDefinitionRepository, "StreamDefinitionRepository must not be null");
+		Assert.notNull(appRegistry, "AppRegistryCommon must not be null");
 		this.streamDefinitionRepository = streamDefinitionRepository;
+		this.appRegistry = appRegistry;
+	}
+
+	public StreamDefinition createStream(String streamName, String dsl, boolean deploy) {
+
```

Comment:
As a general practice, we can avoid adding extra line (though they make it look better) between the statements.

Reply:
which extra line are you referring to? It seems i've added more then one (extra) lines :)

Reply:
Why would we discourage this?  @ilayaperumalg You even point out it makes it look better (more readable).  I know Spring Framework's code style only requires a blank line after method signatures that are multiline, but I'm a fan of it in all cases personally (https://github.com/spring-projects/spring-framework/wiki/Code-Style#blank-lines).

Reply:
@tzolov @mminella I personally like having blank lines when it makes the code look better. I guess I just got accustomed to not having blank lines between the statements inside the method after all those code review comments I received since Spring XD days ;)